resources/l10n/de/LC_MESSAGES/loot.po

```diff
@@ -339,702 +378,648 @@ msgstr ""
-msgstr "Plugins insgesamt"
+#: src/gui/qt/settings/game_tab.cpp:160
+msgid "Local Data Path"
+msgstr "Lokaler Datenpfad"
 
-#: src/gui/html/js/translateStaticText.ts:507
-msgid "General"
-msgstr "Allgemein"
+#: src/gui/qt/settings/game_tab.cpp:162
+msgid "Delete game"
+msgstr "Lösche Videospiel"
 
-#: src/gui/html/js/translateStaticText.ts:511
+#: src/gui/qt/settings/general_tab.cpp:172
 msgid "Default Game"
 msgstr "Standard-Auswahl"
 
-#: src/gui/html/js/translateStaticText.ts:513
-msgid "Autodetect"
-msgstr "Automatisch erkennen"
-
-#: src/gui/html/js/translateStaticText.ts:518
-msgid "Language changes will be applied after LOOT is restarted."
-msgstr "Spracheinstellungen werden wirksam nachdem LOOT neugestartet wird."
-
-#: src/gui/html/js/translateStaticText.ts:522
+#: src/gui/qt/settings/general_tab.cpp:174
 msgid "Theme"
 msgstr "Design"
 
-#: src/gui/html/js/translateStaticText.ts:523
-msgid "Theme changes will be applied after LOOT is restarted."
-msgstr "Designeinstellungen werden wirksam nachdem LOOT neugestartet wird."
+#: src/gui/qt/settings/general_tab.cpp:175
+msgid "Update masterlist before sorting"
+msgstr "Aktualisiere Masterlist vor dem Sortieren"
 
-#: src/gui/html/js/translateStaticText.ts:527
-msgid "default"
-msgstr "Standard"
+#: src/gui/qt/settings/general_tab.cpp:176
+msgid "Check for LOOT updates on startup"
+msgstr "Prüfe auf LOOT Updates beim Programmstart"
 
-#: src/gui/html/js/translateStaticText.ts:532
+#: src/gui/qt/settings/general_tab.cpp:177
 msgid "Enable debug logging"
 msgstr "Aktivere Debug-Protokollierung"
 
-#: src/gui/html/js/translateStaticText.ts:533
+#: src/gui/qt/settings/general_tab.cpp:178
+msgid "Masterlist prelude repository URL"
+msgstr "Masterlist Prelude Repository URL"
+
+#: src/gui/qt/settings/general_tab.cpp:180
+msgid "Masterlist prelude repository branch"
+msgstr "Masterlist Prelude Repository Branch"
+
+#: src/gui/qt/settings/general_tab.cpp:183
+msgid "Language changes will be applied after LOOT is restarted."
+msgstr "Spracheinstellungen werden wirksam nachdem LOOT neugestartet wird."
+
+#: src/gui/qt/settings/general_tab.cpp:185
+msgid "Theme changes will be applied after LOOT is restarted."
+msgstr "Designeinstellungen werden wirksam nachdem LOOT neugestartet wird."
+
+#: src/gui/qt/settings/general_tab.cpp:187
 msgid "The output is logged to the LOOTDebugLog.txt file."
 msgstr "Die Ausgabe wird in die LOOTDebugLog.txt-Datei protokolliert."
 
-#: src/gui/html/js/translateStaticText.ts:538
-msgid "Update masterlist before sorting"
-msgstr "Aktualisiere Masterlist vor dem Sortieren"
+#: src/gui/qt/settings/general_tab.cpp:189
+msgid "A prelude URL is required."
+msgstr "Ein Prelude URL wird benötigt."
 
-#: src/gui/html/js/translateStaticText.ts:541
-msgid "Check for LOOT updates on startup"
-msgstr "Prüfe auf LOOT Updates beim Programmstart"
+#: src/gui/qt/settings/general_tab.cpp:190
+msgid "A prelude branch is required."
+msgstr "Ein Prelude Branch wird benötigt."
```

Comment:
`Eine` instead of `Ein` seems to fit a bit better.

Reply:
Done.